modules/db_dashboard.py

```diff
@@ -2,11 +2,12 @@
 import logging
 from datetime import datetime, timedelta
 import six
+from gluon import current
 
-rslogger = logging.getLogger(settings.logger)
-rslogger.setLevel(settings.log_level)
+rslogger = logging.getLogger(current.settings.logger)
+rslogger.setLevel(current.settings.log_level)
 
-#db.define_table('dash_problem_answers',
+#current.db.define_table('dash_problem_answers',
```

Comment:
Why not just delete these? Are they used?

Reply:
apparently not, since they've been commented out forever.

Reply:
I was actually on the verge of deleting them, but then I decided they they went along with the comment that followed.  That much of what we show in dashboard could be calculated and stored in one of the tables represented by the commented out definitions.

I suppose we could just copy them into an issue somewhere, but they aren't hurting anything where they are either.

Reply:
Sounds good.